packages/text-input-react/src/TextField.tsx

```diff
@@ -39,8 +43,10 @@ export const TextField = ({
             id={id}
             placeholder=" "
             readOnly={readOnly}
+            value={value}
             {...rest}
         />
+        {!value && <span className="jkl-text-field__placeholder">{placeholder}</span>}
```

Comment:
maybe a default value for placeholder and/or a check if placeholder exists before usage?